### PR TITLE
Allow to trigger a retest in Konflux Dashboard

### PR DESCRIPTION
**Pull request by creydr**
Allow to trigger a retest via Konflux Dashboard.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -152,16 +153,27 @@ <h1>Release Build Status</h1>
...
-                                            `
+                                        .map(function (run) {
+
+                                            const parts = run.name.split(" / ")
+                                            const testName = parts[parts.length - 1]
+
+                                            let retestLink = `<a href="javascript:void(0);" onClick="retriggerTest('${item.org}', '${item.repo}', '${item.branch}', '${run.head_sha}', '${testName}')">/retest</a>`
+                                            if (parts.length === 3 && parts[1].endsWith("-ec")) {
+                                                retestLink = "<i title='retest not available on enterprise contract runs'>n/a</i>"
+                                            }
```
Comment (pierDipi) [inline docs/index.html:164]: I wouldn't bother filtering honestly as it couples this to the tests we have

File: docs/index.html [hunk 2]
```diff
@@ -152,16 +153,27 @@ <h1>Release Build Status</h1>
                                 .flatMap(item => {
                                     return item.runs.check_runs
                                         .filter(run => run.name.includes('Konflux'))
-                                        .map(run =>
-                                            `
+                                        .map(function (run) {
+
+                                            const parts = run.name.split(" / ")
+                                            const testName = parts[parts.length - 1]
```
Comment (pierDipi) [inline docs/index.html:159]:
```suggestion
                                            const testName = run.name.slice(run.app.name.length).replace(/[\t /]+/, '').trim()
```

File: docs/index.html [hunk 3]
```diff
@@ -152,16 +153,27 @@ <h1>Release Build Status</h1>
                                 .flatMap(item => {
                                     return item.runs.check_runs
                                         .filter(run => run.name.includes('Konflux'))
-                                        .map(run =>
-                                            `
+                                        .map(function (run) {
```
Comment (pierDipi) [inline docs/index.html:156]:
```suggestion
                                        .map((run) => {
```

File: docs/index.html [hunk 4]
```diff
@@ -152,16 +153,27 @@ <h1>Release Build Status</h1>
...
+
+                                            const parts = run.name.split(" / ")
+                                            const testName = parts[parts.length - 1]
+
+                                            let retestLink = `<a href="javascript:void(0);" onClick="retriggerTest('${item.org}', '${item.repo}', '${item.branch}', '${run.head_sha}', '${testName}')">/retest</a>`
```
Comment (pierDipi) [inline docs/index.html:161]: wondering why this is a link and not a button?

File: docs/index.html (hunk 1 above)
Comment (creydr) [reply docs/index.html]: ok.

Comment (creydr) [reply docs/index.html]: Removed it

File: docs/index.html (hunk 4 above)
Comment (creydr) [reply docs/index.html]: changed it to a button

**Pull request merged**